extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
Okay, then I suggest rewriting that sentence to:

For AMP components, the `height` and `width` attributes are bindable along with the following specific attributes: